packages/client/src/components/Footer.js

```diff
@@ -0,0 +1,59 @@
+import { useContext } from "react";
+import { DiscordLogo, TwitchLogo, TwitterLogo } from "phosphor-react";
+import TextLink from "./TextLink";
+import FooterDiv from "./styled/FooterDiv";
+import IconAnchor from "./styled/IconAnchor";
+import { AuthContext } from "../context/AuthContext";
+
+const Footer = (props) => {
+  const { logout } = useContext(AuthContext);
+
+  return (
+    <FooterDiv>
+      {props.isAdmin && (
+        <div className="linkDiv flex">
+          <TextLink
+            text="Create a post"
+            link="/create-post"
+            margin="0 0 1rem 0"
+          />
+          <TextLink text="Log out" link="/" onClick={() => logout()} />
+        </div>
+      )}
+
+      <div className="subDiv flex">
+        <div className="copyright">
+          <span>©2021 100devs Team</span>
+        </div>
+
+        <div className="iconDiv flex">
+          <IconAnchor
+            href={`https://leonnoel.com/blog/100devs/`}
```

Comment:
Let's keep text input simple when we can:
```suggestion
            href="https://leonnoel.com/blog/100devs/"
```